src/java/org/apache/cassandra/cql3/selection/SelectorFactories.java

```diff
@@ -165,6 +171,17 @@ public boolean containsWritetimeSelectorFactory()
         return containsWritetimeFactory;
     }
 
+    /**
+     * Checks if this <code>SelectorFactories</code> contains at least one factory for maxWritetime selectors.
+     *
+     * @return <code>true</code> if this <code>SelectorFactories</code> contains at least one factory for maxWritetime
+     * selectors, <code>false</code> otherwise.
```

Comment:
```suggestion
     * <code>true</code> if this <code>SelectorFactories</code> contains at least one factory for maxWritetime
     * selectors, <code>false</code> otherwise.
     * @return {@link true} if this {@link SelectorFactories} contains at least one factory for maxWritetime
     * selectors, {@link false} otherwise.
```